apollo-router/src/spec/schema.rs

```diff
@@ -84,7 +85,14 @@ impl Schema {
 
         let mut api_schema: Option<_> = None;
         let mut connectors: Option<_> = None;
-        let expansion = expand_connectors(sdl).map_err(SchemaError::Connector)?;
+        let connectors_config = config
+            .apollo_plugins
+            .plugins
+            .get("preview_connectors")
```

Comment:
should we make this a const so it's easier to change when we drop "preview_"?

Reply:
Probably, I wasn't sure how this gets set in the first place—where does this config get defined? Is it related to the plugin name, and can that also be a const?

Reply:
yeah it's the plugin name as registered, but i'm guessing that can't be a const: https://github.com/apollographql/router/blob/connectors/apollo-router/src/plugins/connectors/plugin.rs#L120